DEPLOYMENT_CHECKLIST.md

```diff
@@ -1,108 +1,74 @@
-
-### Subgraph
-
-- [ ] Update `packages/subgraph/manifest/data/NETWORK.json` where `NETWORK` is replaced with the deployed network with the new contract addresses. If the file doesn't exist create a new one.
-- [ ] Update the version in `packages/subgraph/package.json`
-- [ ] Update `packages/subgraph/.env` with the correct values
-  - [ ] set `NETWORK_NAME` to the deployed network
-  - [ ] set `SUBGRAPH_NAME` to `osx`
-  - [ ] set `GRAPH_KEY` with the value obtained from the [Satsuma Dashboard](https://app.satsuma.xyz/dashboard)
-  - [ ] set the `SUBGRAPH_VERSION` to the same value as in `packages/subgraph/package.json`
-- [ ] Run `yarn manifest` in `packages/subgraph` to generate the manifest
-- [ ] Run `yarn build` in `packages/subgraph` to build the subgraph
-- [ ] Run `yarn test` in `packages/subgraph` to test the subgraph
-- [ ] Run `yarn deploy` in `packages/subgraph` to deploy the subgraph
-- [ ] Test the new deployed subgraph with the frontend team
-- [ ] Promote the new subgraph to live in the [Satsuma Dashboard](https://app.satsuma.xyz/dashboard)
-
-## Appendix
-
-- Changing the owner of the chosen ENS domains will also revoke the permissions of the `DAO_ENSSubdomainRegistrar` and `Plugin_ENSSubdomainRegistrar`. Therefore if the ownership gets transfered, restore the approval for these 2 contracts.
+- [ ] Go to `packages/contracts/networks.json` and add your custom network to which you want to deploy to.
+
+  If your chain doesn’t support contract verification:
+
+  - Make sure that whatever custom network you add into this file, its `deploy` key looks exactly: <br>
+    `"deploy": ["./deploy/new"]`
+
+  If your chain supports contract verification:
+
+  - set `deploy` key such as: <br>
+    `"deploy": ["./deploy/new", "./deploy/verification"]` .
+    Not only that, you will need to provide the API_KEY or rpc url for contract verification, depending on the chain. Follow the guide https://hardhat.org/hardhat-runner/plugins/nomicfoundation-hardhat-verify in such case.
+
+- [ ] Management DAO is the dao that will govern the protocol/framework and its rules. So go to `packages/contracts/deploy/management-dao-metadata.json` and update its values as you wish. This is deployed to the IPFS and its cid is stored in the managing dao so users can get information about what this managing dao does.
+- [ ] Ensure to add `ETH_KEY` in the `.env` which is your private key that will be used to deploy all the contracts. Example:
+  ```jsx
+  ETH_KEY = YOUR_PRIVATE_KEY; // without `0x` prefix
+  ```
+- [ ] Our contracts use ENS, so it’s important that you safely add the following ENS names in the packages/contracts/.env file. <br>
+
+  - `NETWORK_DAO_ENS_DOMAIN`
+  - `NETWORK_PLUGIN_ENS_DOMAIN`
+
+    NOTE that `NETWORK` must be replaced to the network name you’re trying to deploy to.
+    Example would be:
+
+    ```jsx
+    SEPOLIA_DAO_ENS_DOMAIN = testdao.eth;
+    SEPOLIA_PLUGIN_ENS_DOMAIN = testplugin.eth;
```

Comment:
```suggestion
    ```jsx
    SEPOLIA_DAO_ENS_DOMAIN = testdao.eth;
    SEPOLIA_PLUGIN_ENS_DOMAIN = testplugin.eth;
```
```suggestion
    ```
    SEPOLIA_DAO_ENS_DOMAIN="testdao.eth"
    SEPOLIA_PLUGIN_ENS_DOMAIN="testplugin.eth"
```
Please make sure that the checklist suggestions are correct.